contracts/LoanManager.sol

```diff
@@ -125,6 +140,16 @@ contract LoanManager is Restricted, TokenReceiver {
         emit NewLoan(productId, loanId, msg.sender, msg.value, loanAmount, repaymentAmount, maturity);
     }
 
+    function addExtraCollateral(uint loanId) external payable {
+        require(loanId < loans.length, "invalid loanId");
+        LoanData storage loan = loans[loanId];
+        require(loan.state == LoanState.Open, "loan state must be Open");
```

Comment:
Do we want to allow adding collateral to a loan which is already defaulted (i.e. not repaid until maturity). 
It doesn't do too much harm as the collection will refund any leftover. 
If that's the case then we should be explicit in a comment about it. Alternatively, add an extra require for maturity.

Reply:
I've no strong opinion on that.